Package.swift

```diff
@@ -16,17 +16,19 @@ let package = Package(
     ],
     products: [
         .library(
-            name: "SwiftCache",
-            targets: ["SwiftCache"]
+            name: "ResourceProvider",
+            targets: ["ResourceProvider"]
         )
     ],
     targets: [
         .target(
-            name: "SwiftCache"
+            name: "ResourceProvider"
         ),
         .testTarget(
-            name: "SwiftCacheTests",
-            dependencies: ["SwiftCache"]
+            name: "ResourceProviderTests",
+            dependencies: ["ResourceProvider"],
+            resources: [.process("Resources")]
         )
-    ]
+    ],
+    swiftLanguageModes: [.v5, .v6]
```

Comment:
I don't think you need this

Reply:
I'm starting to realize I haven't yet figured out what _this_ is exactly _for_. Meanwhile I'll take your word for it and remove it.